cfgov/jinja2/v1/_includes/molecules/contact-hyperlink.html

```diff
@@ -0,0 +1,28 @@
+{# ==========================================================================
+
+   Contact Hyperlink
+
+   ==========================================================================
+
+   Description:
+
+   Create a Contact Hyperlink molecule when given:
+
+   value:                 ContactHyperlink block StreamValue.
+
+   value.url:             Hyperlink URL.
+
+   value.text:            Hyperlink text (optional).
+
+   ========================================================================== #}
+<div class="m-contact m-contact-hyperlink" data-qa-hook="contact-hyperlink">
```

Comment:
I think `data-qa-hook="contact-hyperlink"` is unused.

Reply:
Yeah, the corresponding thing in `contact-email.html` could also be removed.